helm/hpcc/values.schema.json

```diff
@@ -2133,6 +2133,11 @@
           "type": "boolean",
           "description": "[Optional] If lingerPeriod set, allows Thor to process more graphs from any job"
         },
+        "thorConnectTimeout": {
```

Comment:
Should you add a commented-out entry in the values.yaml too, by way of documentation?

Reply:
@richardkchapman - have renamed default property to 'maxGraphStartupTime' which makes more sense in k8s property value under 'thor:', but it will fallback to checking for legacy property 'thorConnectTimeout'
Also added commented-out example in stock values.yaml